### PR TITLE
#115852233 Added home logo with link and brand colors

### PR DESCRIPTION
**Pull request by GLBaird**
Added home logo to design with brand colours, and added a responsive
mobile view.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,16 +18,12 @@
...
-
-					<a class="navigation-title" href="">
-						
-						<img class="img" src="https://raw.githubusercontent.com/pyupio/statuspage/master/template/logo.png" height="30">
-						<h1 class="title">Status</h1>
-					</a>
-
-
+				<a href="https://www.autoenrolment.co.uk" title="Auto Enrolment Home Page" target="_blank">
+					<img class="img" src="https://d1kpndw70yxcc7.cloudfront.net/images/files/000/000/001/optimized/logo.png" alt="Smart Pension">
```
Comment (alaibe) [inline index.html]: upload the logo to the github repository

File: style.css [hunk 2]
```diff
@@ -82,29 +82,28 @@ img {
...
 	font-family: 'Gotham Rounded A', 'Gotham Rounded B', 'Helvetica Neue', Arial, sans-serif;
 	line-height: 5.2rem;
-	font-size: 1.6rem;
-	padding: 0;
-	position: relative;
+	font-size: 2.2rem;
 	text-decoration: none;
+    position: relative;
+    margin: 1em 0 0 0;
+    color: #77ba9c; /* Secondary color */
```
Comment (alaibe) [inline style.css]: no need for comment

File: style.css [hunk 3]
```diff
@@ -82,29 +82,28 @@ img {
 	background: #f4f5f6;
```
Comment (alaibe) [inline style.css]: indentation is broken in this file it seems?

Comment (GLBaird) [inline style.css]: It looks okay in my IDE. I only added a few lines, would you like me to auto-reformat?

Comment (GLBaird) [inline style.css]: Just auto formatted it, and only noticed when I looked at it on GITHub that it made the inline font text-wrap! Might be best to undo that, as I did not realise the font was inline (My IDE was hiding that data from me).

**Comment by alaibe** [comment]
:+1:

**Pull request merged**